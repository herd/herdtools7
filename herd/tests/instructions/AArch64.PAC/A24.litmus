AArch64 A24
Variant=pauth2
{
  0:x0=x; 0:x1=x; int64_t 0:x2=0;
}
P0;
pacdza x0;
autdza x1;
cmp x0,x1;
b.ne end;
mov x2,#1;
end:;
forall (0:x2=1)
