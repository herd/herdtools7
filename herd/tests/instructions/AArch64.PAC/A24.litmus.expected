Test A24 Required
States 2
0:X2=1;
Ok
Witnesses
Positive: 2 Negative: 0
Condition forall (0:X2=1)
Observation A24 Always 2 0
Hash=c3e87fb8200e9b6605c53e3cb06857df

